### DISCUSSION TITLE
Welcome to devsrequests Discussions!

### DISCUSSION THREAD

**Discussion by JOS-RE** [devstrons/devsrequests#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect the participants with the community. We hope that you:
  *  ✔ Make sure you have registered to the Event
  * ❓ Ask questions you’re wondering about.
  * 👍 Share ideas.
  * 💬 Engage with other community members.
  * 🌐 Welcome and help each other to make this event a success 💪.
  * 👨🏻‍🏫  You can avail help from our list of [Mentors](https://github.com/devstrons/devsrequests/blob/main/MENTORS.md)

➡️  To get started, comment below with an introduction of yourself and tell us how we can help you :D

**Comment by TheInspiredConjurer** [comment]
Hello!

I am Ramyak Rohan Mohanty, a final year MCA student from Odisha. I am an aspiring web developer and have created some basic projects in HTML and CSS.

Languages I am currently working in HTML5, CSS3, JavaScript, Python-Django and SQL.

I am also very passionate about Open Source and believe in helping others when and however I can.

I hope I will get the necessary guidance and mentorship to increase my skills.

Thank you.

Here is my linkedin profile: https://www.linkedin.com/in/rrm1995/
you can reach me via mail (mail ID:- ramyakrohanmohanty@gmail.com) or Discord (ID: The Inspired Conjurer#8271)

**Reply by JOS-RE** [reply]
Glad to have you here, feel free to get your doubts clarified 😄
